### PR TITLE
Update KiriKiri Engine

### PR DESCRIPTION
**Pull request by MichaelHinrichs**
### SteamDB app page links to a few games using this
https://steamdb.info/app/1543330

### Brief explanation of the change
Find KiriKiri plugins, in case the engine is not detected from data.xp3.

### CODE REVIEW

**Comment by SteamTracker** [comment]
After running changed rules in this pull request *(this bot only tests rules.ini changes)*:

Changed technologies: **Engine.KiriKiri**

**374** apps still match.

These apps now match: **(8)** [572140](https://steamdb.info/app/572140/depots/) [863870](https://steamdb.info/app/863870/depots/) [869870](https://steamdb.info/app/869870/depots/) [1180360](https://steamdb.info/app/1180360/depots/) [1274780](https://steamdb.info/app/1274780/depots/) [1543330](https://steamdb.info/app/1543330/depots/) [2555930](https://steamdb.info/app/2555930/depots/) [2581160](https://steamdb.info/app/2581160/depots/)

Please confirm that all these changed apps are correct.

File: rules.ini [hunk 1]
```diff
@@ -123,7 +123,8 @@ Irrlicht = (?:^|/)Irrlicht\.dll$
 IW_Engine[] = \.iwd$
 IW_Engine[] = (?:common(?:_zm|_mp)?|code_post_gfx)\.ff$
 Kex = \.kpf$
-KiriKiri = (?:^|/)data\.xp3$
+KiriKiri[] = (?:^|/)data\.xp3$
+KiriKiri[] = ^plugin/kr(?:movie|flash|krsteam)\.dll$
```
Comment (Lifeismana) [inline rules.ini:127]:
```suggestion
KiriKiri[] = (?:^|/)krflash\.dll$
```

**Comment by SteamDB-Tracker** [comment]
After running changed rules in this pull request *(this bot only tests rules.ini changes)*:

Changed technologies: **Engine.KiriKiri**

**382** apps still match.

These apps now match: **(8)** [572140](https://steamdb.info/app/572140/depots/) [863870](https://steamdb.info/app/863870/depots/) [869870](https://steamdb.info/app/869870/depots/) [1180360](https://steamdb.info/app/1180360/depots/) [1543330](https://steamdb.info/app/1543330/depots/) [2555930](https://steamdb.info/app/2555930/depots/) [2581160](https://steamdb.info/app/2581160/depots/) [2862530](https://steamdb.info/app/2862530/depots/)

Please confirm that all these changed apps are correct.

**Comment by Lifeismana** [comment]
merged in 6c38740a8a90f4413d50605e896ccdfa321df9d5

**Pull request closed without merging**